src/Commands/Configure.ts

```diff
@@ -45,11 +35,25 @@ export class Configure implements CommandModule {
         describe: `Enables network tests only`
       })
       .middleware((args: Arguments) => {
+        const { api, bus } = Helpers.getClusterUrls(args);
+
         container.register(ConnectivityUrls, {
           useValue: new Map(
             Object.values(TestType).map((type: TestType) => {
               try {
-                return [type, new URL(args[type] as string)];
+                switch (type) {
```

Comment:
I don't see any reason to use `switch` statements within `map`, you should use either `switch` or `Object.values(TestType).map()`